### PR TITLE
Add Plugin extension

### PR DESCRIPTION
**Pull request by danigm**
This new extension point will allow to extend the kolibri app with external flatpaks that just provides new extensions as python modules without the need to update the base app.

This is an example plugin manifest: https://github.com/danigm/org.learningequality.Kolibri.Plugin.Explore/blob/master/org.learningequality.Kolibri.Plugin.Explore.json

### CODE REVIEW

**Review by manuq — APPROVED** [review]
This looks good to me. @dylanmccall do you want to review it as well?

**Comment by dylanmccall** [comment]
Yeah, I'd like to review this please. Will do in a couple hours :)

File: org.learningequality.Kolibri.yaml
Comment (dylanmccall) [inline org.learningequality.Kolibri.yaml:29]: One thing to bear in mind here is we'll need to switch to python3.8 at the same time as we switch to the org.gnome.Platform//3.38 runtime, and I expect we'll want to do that fairly soon. Fine for now since we're building the only plugin, but I guess we'd need to either increment the version of this extension point, or keep "/app/lib/python3.7/..." in our PYTHONPATH. Does that sound okay, or should we instead build plugins so they use a generic looking site-packages path?

Gajim, for instance has plugins install to `/app/plugins/${EXTENSION_NAME}/site-packages`: https://github.com/flathub/org.gajim.Gajim/blob/master/org.gajim.Gajim.yaml

**Review by dylanmccall — CHANGES_REQUESTED** [review]
Okay, I went through this and tried building a plugin. Seems like a good approach :) I have one quick fix, and one question.

File: org.learningequality.Kolibri.yaml
Comment (danigm) [reply org.learningequality.Kolibri.yaml]: Yes, you're right, it's better to use the same tag. I've updated this.

File: org.learningequality.Kolibri.yaml
Comment (danigm) [reply org.learningequality.Kolibri.yaml]: We've the version number in the `Plugin`, so we can update that when the python version changes. In any case, I've updated the plugin to use a path without the python version.

**Comment by dylanmccall** [comment]
Great! This looks good to me now :) I'll hold off on merging until the Kolibri daemon stuff is ready as well, then we can put it all in the same release.

**Comment by danigm** [comment]
I think this get closed by mistake when the PR #17 gets merged, can we reopen this PR?

**Comment by danigm** [comment]
@dylanmccall can you take a look to this? I think that you closes this PR after merge the #17, and uses this ID instead an issue ID.

**Comment by dylanmccall** [comment]
Oh, blah, I forgot to force-push the new commits when I merged your change into that pull request :( May you please rebase this on the current master? Indeed, we need to re-open this one. @jamalex ?

**Pull request closed without merging**
